moveit_planners/chomp/chomp_interface/include/chomp_interface/chomp_interface.h

```diff
@@ -50,7 +50,7 @@ class CHOMPInterface : public chomp::ChompPlanner
 public:
   CHOMPInterface(const ros::NodeHandle& nh = ros::NodeHandle("~"));
 
-  const chomp::ChompParameters& getParams() const
+  chomp::ChompParameters& getParams()
```

Comment:
this should remain const for best practice. if you want to change the params you can create a ``setParams(params)``, or another approach you'll likely prefer making a ``getParamsNonConst()``

Reply:
changed back to const, currently using a getter method to get a [non constant version](https://github.com/ros-planning/moveit/pull/987/commits/2ce9804a47f76324fb2e8babe09be5b7e609e582#diff-c45064b5ae236917541fdeca966527cbR77) of the ChompParameters